tests/conftest.py

```diff
@@ -27,6 +27,17 @@ def nop(*args, **kwargs):
     monkeypatch.setattr("pyspark.sql.SparkSession.stop", nop)
 
 
+@pytest.fixture
+def df_equals(row_to_dict):
+    def to_comparable(df):
+        return sorted(map(row_to_dict, df.collect()))
```

Comment:
`sorted` is being called on a `list` of `dicts` where each element of the list represents one row of a dataframe.